packages/sn-filter-pane/src/ext/property-panel/data/text-align-items.ts

```diff
@@ -46,7 +46,8 @@ const textAlignItems = {
       },
     ],
     defaultValue: 'left',
-    show(data) {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    show(data: EngineAPI.IGenericListProperties & any) {
```

Comment:
Can we use `unknown` to avoid the `eslint-disable…` or will it complain then?

Reply:
I added a new interface with the missing properties instead